test/katex-spec.js

```diff
@@ -2717,27 +2717,10 @@ describe("href and url commands", function() {
     });
 
     it("should be marked up correctly", function() {
-        const markup = katex.renderToString(r`\href{http://example.com/}{example here}`);
+        const markup = katex.renderToString(r`\href{http://example.com/}{example here}`, {trust: true});
         expect(markup).toContain("<a href=\"http://example.com/\">");
     });
 
-    it("should allow protocols in allowedProtocols", function() {
-        expect("\\href{relative}{foo}").toParse();
-        expect("\\href{ftp://x}{foo}").toParse(new Settings({
-            allowedProtocols: ["ftp"],
-        }));
-        expect("\\href{ftp://x}{foo}").toParse(new Settings({
-            allowedProtocols: ["*"],
-        }));
-    });
-
-    it("should not allow protocols not in allowedProtocols", function() {
-        expect("\\href{javascript:alert('x')}{foo}").not.toParse();
-        expect("\\href{relative}{foo}").not.toParse(new Settings({
-            allowedProtocols: [],
-        }));
-    });
-
```

Comment:
It would be nice to replace these with tests of the corresponding `trust` settings (as in the documentation examples). In general, it would be good to add `trust` tests. Because `trust` violations don't throw errors, you could probably just use snapshot tests... Or test for the appropriate parse node.

Reply:
Good idea.  Will do.

Reply:
I've re-added the tests but I had to refactor them a bit to be snapshot tests since `\href` with `trust: false` parses, but produces a different parse tree from when `trust: true` is set.

I've also set the default for the unit tests to `trust: false`.  I can probably remove that setting though since `false` is the default.